### PR TITLE
Update github-actions in this repository

### PR DESCRIPTION
**Pull request by at-wat**
Avoid opening Renovate PRs in each repository.

### CODE REVIEW

**Review by daenney — APPROVED** [review]

File: renovate.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "commitBody": "Generated by renovateBot",
+  "github-actions": {
+    "fileMatch": ["^ci\/workflows\/[^/]+\.ya?ml$"]
```
Comment (at-wat) [inline renovate.json:7]:
```suggestion
    "fileMatch": ["^ci\\/workflows\\/[^/]+\\.ya?ml$"]
```
?

Comment (daenney) [reply renovate.json:7]: Honestly, idk. Do they have an example in their docs somewhere?

Comment (at-wat) [reply renovate.json:7]: In JSON's spec, updated one should be correct.

Comment (at-wat) [reply renovate.json:7]: With updated one, `cat renovate.json | jq -r '.["github-actions"].fileMatch[0]'` returns `^ci\/workflows\/[^/]+\.ya?ml$`, so, this must be correct.

**Pull request merged**